packages/url_launcher/url_launcher/README.md

```diff
@@ -49,6 +49,36 @@ void _launchURL() async =>
     await canLaunch(_url) ? await launch(_url) : throw 'Could not launch $_url';
 ```
 
+### Android
+
+Starting from API 30 Android requires package visibility configuration in your
+`AndroidManifest.xml` otherwise `canLaunch` would return `false`. A `<queries>`
+element could be added to your manifest as a child of the root element. The
+snippet below shows a variation, feel free to tailor it to your needs. The
```

Comment:
How about replacing this and the next two sentences with:

"The snippet below shows an example for an application that uses `https`, `tel`, and `mailto` URLs with `url_launcher`. See \[the Android documentation\](https://developer.android.com/training/package-visibility/use-cases) for examples of other queries."

(Saying "feel free to tailor it" seems misleading to me; tailoring a permission list is not a "do it it if you feel like it" task, it's something that must be done. And since the example app's queries list is a subset of the example here, I don't think it helps to point people there.)

Reply:
The https://developer.android.com/training/package-visibility/use-cases is a little different than https://developer.android.com/training/basics/intents/package-visibility which is in the `can Launch` documentation in url_launcher.dart. Should we change that too?